tests/test_circuit_validation.py

```diff
@@ -1,37 +1,34 @@
 try:
-    from unittest.mock import patch
+    from unittest.mock import Mock, patch
```

Comment:
can we just use the unittest ones now? or is our lower bound python still too old?

Reply:
I'll look into it